fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/Operator.java

```diff
@@ -40,6 +46,10 @@ public abstract class Operator {
 
     protected RowOutputInfo rowOutputInfo;
 
+    // Add salt make the original equivalent operators nonequivalent to avoid Group
+    // mutual reference in Memo
+    protected long salt = 0;
```

Comment:
I don't this it's  reasonable.

Reply:
Only LogicalScanOperator/PhysicalScanOperator orginates from CboTablePruneRule has salt